libs/mutantschemata/cpp_source/api.cpp

```diff
@@ -48,19 +49,8 @@ void runSchemataCpp(SchemataApiCpp *sac, CppString::CppStr cs, CppString::CppStr
 
     // call setup of clang
     setupClang(argv.size() - 1, argv.data(), sac);
+}
 
-    // Simple demonstration of how to create a SchemataMutant, set the variables and insert in database
-    // NOTE: apiBuildMutant needs to be called (ONCE) in order to make sure the db has created the table
-    /*
-    CppType::SchemataMutant sm;
-    sm.mut_id = 500;
-    sm.loc = {4,5};
-    sm.offset = {0,3};
-    sm.status = 0;
-    sac->apiBuildMutant();
-    sac->apiInsertSchemataMutant(sm);
-    sm.mut_id = 365;
-    sm.status = 3;
-    sac->apiInsertSchemataMutant(sm);
-    */
+int setEnvironmentVariable(CppString::CppStr variable, CppString::CppStr value){
+  return setenv(variable.cppStr->c_str(), value.cppStr->c_str(), 1);
```

Comment:
note: If the D side is driving the mutated binary then it is probably the call to std.process.spawnProcess that you want to inject the environment variables through.